src/WhereConstraints/SQLOperator.php

```diff
@@ -14,20 +14,48 @@ public static function missingValueForColumn(string $column): string
     public function enumDefinition(): string
     {
         return /** @lang GraphQL */ <<<'GRAPHQL'
+"The available SQL operators that are used to filter query results."
 enum SQLOperator {
+    "Equal operator (=)"
```

Comment:
```suggestion
    "Equal operator (`=`)."
```

Let's wrap the SQL-equivalents in backticks. GraphQL IDE's such as Playground format descriptions as Markdown.

Reply:
Also done :+1: